aten/src/ATen/native/native_functions.yaml

```diff
@@ -1188,6 +1188,15 @@
 - func: mvlgamma_(Tensor self, int64_t p) -> Tensor
   variants: method
 
+- func: narrow_copy(Tensor self, int64_t dim, int64_t start, int64_t length) -> Tensor
+  variants: method
+  dispatch:
```

Comment:
I had to specifically dispatch for ```_get_sparse_impl``` to work correctly.  There is probably also a case to be made to not dispatch at all for dense tensors.